torch/distributions/transformed_distribution.py

```diff
@@ -87,21 +88,35 @@ def log_prob(self, value):
                                    event_dim - len(self.base_dist.event_shape))
         return log_prob
 
+    def _monotonize_cdf(self, value):
+        """
+        This conditionally flips ``value -> 1-value`` to ensure :meth:`cdf` is
+        monotone increasing.
+        """
+        sign = 1
+        for transform in self.transforms:
+            sign = sign * transform.sign
```

Comment:
Should we cache this? We could use a `ComposeTransform` to avoid looping inside methods of this distribution.

Reply:
The current list-based implementation works, so I'd rather focus effort on bugfixes :smile: